### ISSUE TITLE
Dragging a collapsed function call block causes a crash if the function has inputs

### ISSUE THREAD

**Issue by nederland074** [google/blockly#3589]
**Describe the bug**

Collapsed function call blocks cannot be dragged because of crashes. This is reproduced if the corresponding function definition block has at least one input. It works as expected if a function has no inputs, though.

**To Reproduce**

1. Drag out a function definition block.
2. Add an input to the block.
3. Drag out the corresponding function call block.
4. Collapse the function call block via Right Click -> "collapse block".
5. Drag the function call block.

**Expected behavior**

Dragging should work without crashes.

**Stack Traces**

```
Uncaught TypeError: Cannot read property 'fieldRow' of undefined
    at Blockly.InsertionMarkerManager.createMarkerBlock_ (blockly_compressed.js:146)
    at new Blockly.InsertionMarkerManager (blockly_compressed.js:141)
    at new Blockly.BlockDragger (blockly_compressed.js:161)
    at Blockly.TouchGesture.Blockly.Gesture.startDraggingBlock_ (blockly_compressed.js:303)
    at Blockly.TouchGesture.Blockly.Gesture.updateIsDraggingBlock_ (blockly_compressed.js:300)
    at Blockly.TouchGesture.Blockly.Gesture.updateIsDragging_ (blockly_compressed.js:302)
    at Blockly.TouchGesture.Blockly.Gesture.updateFromEvent_ (blockly_compressed.js:297)
    at Blockly.TouchGesture.Blockly.Gesture.handleMove (blockly_compressed.js:308)
    at Blockly.TouchGesture.handleMove (blockly_compressed.js:590)
    at HTMLDocument.h (blockly_compressed.js:782)
```

**Additional context**

This issue looks similar to [#2363](https://github.com/google/blockly/issues/2363).  I tried a workaround suggested here: [https://github.com/google/blockly/issues/2363#issuecomment-502025753](https://github.com/google/blockly/issues/2363#issuecomment-502025753) and it seems to work fine.

**Comment by nederland074** [comment]
also here's a link to a test case: [https://blockly-demo.appspot.com/static/demos/code/index.html#86em5q](https://blockly-demo.appspot.com/static/demos/code/index.html#86em5q)
if you try to drag the collapsed block you can see crashes in the console

**Comment by alschmiedt** [comment]
Thanks for reporting @nederland074! Should be fixed in #3591.
